/teleop:
  ros__parameters:
    frequency: 10  # set to 0, to disable manual flight modes
    uri1: "radio://*/80/2M/E7C2C2C208?safelink=0&autoping=0"
    uri2: "radio://*/80/2M/E7C2C2C20A?safelink=0&autoping=0"
    uri3: "radio://*/80/2M/E7C2C2C207?safelink=0&autoping=0"
    uri4: "radio://*/80/2M/E7C2C2C2C2?safelink=0&autoping=0"
    joy1:
      yawrate:
        axis: 3
        max: 20.0 # deg/s
        deadband: 0.0 # deg/s
      thrust:
        axis: 2
        max: 20000.0 # PWM
        deadband: 0.0
    joy2:
      yawrate:
        axis: 3
        max: 20.0 # deg/s
        deadband: 0.0 # deg/s
      thrust:
        axis: 2
        max: 20000.0 # PWM
        deadband: 0.0
    joy3:
      yawrate:
        axis: 3
        max: 20.0 # deg/s
        deadband: 0.0 # deg/s
      thrust:
        axis: 2
        max: 20000.0 # PWM
        deadband: 0.0
    joy4:
      yawrate:
        axis: 3
        max: 20.0 # deg/s
        deadband: 0.0 # deg/s
      thrust:
        axis: 2
        max: 20000.0 # PWM
        deadband: 0.0


